### ISSUE TITLE
Brotli Compression

### ISSUE THREAD

**Issue by Able2Code** [jwilder/nginx-proxy#1201]
Hi, it's possible to add Brotli compression to the docker image ?

**Comment by bfayers** [comment]
Do you mean ngx_brotli? https://github.com/google/ngx_brotli

**Comment by EmixMaxime** [comment]
Yes to add brotli compression you have to use ngx_brotli or a fork like https://github.com/eustas/ngx_brotli. I've the same question.

**Comment by chiqui3d** [comment]
Yes please.
